DS/DS2/MainStackImplementation.java

```diff
@@ -0,0 +1,77 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+//class used to define a stack and its operation
+class Node {
+	Node addressOfNextNode,top;
+	Object listInformation;
+}
+class StackUsingLinkedList extends Node{
```

Comment:
ACTION: Why are you extending node class, you don't extend Node class to create a instance of it.